stdlib/multiprocessing/synchronize.pyi

```diff
@@ -28,8 +29,11 @@ class Condition(AbstractContextManager[bool]):
     def wait_for(self, predicate: Callable[[], bool], timeout: float | None = ...) -> bool: ...
     def acquire(self, block: bool = ..., timeout: float | None = ...) -> bool: ...
     def release(self) -> None: ...
+    def __exit__(
+        self, exc_type: type[BaseException] | None, exc_val: BaseException | None, exc_tb: TracebackType | None
+    ) -> bool | None: ...
```

Comment:
```suggestion
    def __exit__(
        self, __exc_type: type[BaseException] | None, __exc_val: BaseException | None, __exc_tb: TracebackType | None
    ) -> None: ...
```
It takes *args at runtime, but those ultimately get forwarded to `_multiprocessing.SemLock.__exit__` (defined in semaphore.c), which does take exactly three arguments.

It always returns None unless you pass in a very exotic kind of lock.